kview/src/main/java/dev/ikm/komet/kview/mvvm/view/descriptionname/DescriptionNameController.java

```diff
@@ -103,47 +107,52 @@ public DescriptionNameController() { }
 
     @FXML
     public void initialize() {
+        ChangeListener fieldsValidationListner = (obs, oldValue, newValue) -> {
+            descrNameViewModel.validate();
+            descrNameViewModel.setPropertyValue(IS_INVALID, descrNameViewModel.hasErrorMsgs());
+        };
+
+        submitButton.disableProperty().bind(descrNameViewModel.getProperty(IS_INVALID));
 
         populateDialectComboBoxes();
 
+        editDescriptionTitleLabel.textProperty().bind(descrNameViewModel.getProperty(TITLE_TEXT));
+
         //TODO These are temp hard coded values:
         // Can use below code later?
         // setupComboBox(nameDescriptionType, fetchDescendentsOfConcept(getViewProperties(), TinkarTerm.DESCRIPTION_TYPE.publicId()));
 
         setupComboBox(nameDescriptionType, fetchDescriptionTypes()); // Hard coded
-        nameDescriptionType.valueProperty().bind(descrNameViewModel.getProperty(NAME_TYPE));
+        ObjectProperty<ConceptEntity> nameTypeProp = descrNameViewModel.getProperty(NAME_TYPE);
+        nameDescriptionType.valueProperty().bind(nameTypeProp);
+        nameTypeProp.addListener(fieldsValidationListner);
 
-        editDescriptionTitleLabel.textProperty().bind(descrNameViewModel.getProperty(TITLE_TEXT));
-        nameTextField.textProperty().bindBidirectional(descrNameViewModel.getProperty(NAME_TEXT));
+        SimpleStringProperty nameTextProp = descrNameViewModel.getProperty(NAME_TEXT);
```

Comment:
Can the statement be more abstract? 
`StringProperty nameTextProp = ...;`

Reply:
fixed.